### PR TITLE
fix(customerStats): Fix not correct stacked chart - [TET-138] 

### PR DESCRIPTION
**Pull request by priscilawebdev**
related to https://github.com/getsentry/getsentry/pull/7860

Requires https://github.com/getsentry/getsentry/pull/7860

### CODE REVIEW

**Review by matejminar — COMMENTED** [review]
I'm seeing some outcomes multiple times, can you repro?
I can send you the payload.

image

File: static/app/types/echarts.tsx [hunk 1]
```diff
@@ -103,6 +103,11 @@ export type EChartDataZoomHandler = EChartEventHandler<{
   startValue?: number;
 }>;
 
+export type DataPoint = {
```
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply static/app/types/echarts.tsx:106]: we could, but the name can be `undefined` and I was not sure its exactly the same thing

**Comment by priscilawebdev** [comment]
```
I'm seeing some outcomes multiple times, can you repro?
I can send you the payload.
```

Thanks, Matej! This shall be fixed now

File: static/app/types/echarts.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply static/app/types/echarts.tsx]: I think there is no harm, so I have applied the feedback. thanks

**Review by matejminar — APPROVED** [review]

**Pull request merged**